fake_ubersmith/api/methods/uber.py

```diff
@@ -40,7 +40,11 @@ def hook_to(self, entity):
     def check_login(self, form_data):
         for client in self.data_store.clients:
             if client['uber_login'] == form_data['login'] and client['uber_pass'] == form_data['pass']:
-                return response(data={"client_id": client['clientid']})
+                return response(data={"client_id": client['clientid'], "contact_id": client["contact_id"]})
```

Comment:
rather than using these for loops, it would be more functional (and in my opinion better) to do something like this:

```python
client = next((c for c in self.datastore.clients if c['uber_login'] == form_data['login'] and c['uber_pass' = form_data['pass']), None)
if not client:
    client = next(c for c in self.datastore.contacts if ...)
return response(data=...)
```

That way there's only one return in the function and less branches in the code. An alternative would be to pass the second `next(...)` in the default value spot of the first `next(...)` instead of `None`, that way we don't even need the `if not client`